ts/parts/Tooltip.ts

```diff
@@ -1033,7 +1033,7 @@ class Tooltip {
         delay = pick(delay, this.options.hideDelay, 500);
         if (!this.isHidden) {
             this.hideTimer = syncTimeout(function (): void {
-                tooltip.getLabel()[delay ? 'fadeOut' : 'hide']();
+                tooltip.getLabel().fadeOut();
```

Comment:
So now fadeOut animation for about 150ms is unavoidable even when setting delay to 0?
Then the comment above this function should reflect that - `A value of 0 disables the fade out animation.` will no longer be true.

Reply:
Right, I didn't notice the comment - thanks for pointing that out. On the other hand: is there a difference in current implementation between `hideDelay: 0` & `hideDelay: 1`: https://jsfiddle.net/BlackLabel/xevg84r6/
I can't spot the difference. Or am I missing something?

Reply:
The `fadeOut` function takes one argument, `duration`. So I think we should pass a duration of 0 if delay is 0:

`tooltip.getLabel().fadeOut(delay ? void 0 : 0);`

Then we need to fix the `fadeOut` function so it accepts 0, which is a bug in any case:

`duration: pick(duration, 150),`